UD1xh4-YyjT2e/cod-a3V3DV_UD1xh4-YyjT2e.xml

```diff
@@ -71,7 +79,40 @@
           <lb ed="#Z"/>in cogitationem uenire; sicut fumo viso, ignem subesse credimus</quote>
         </p>
         <p>
-          <lb ed="#Z"/>Testline.
+          <lb ed="#Z"/>Distinguit <ref>idem ille</ref> signa, in <seg>signa naturalia et data 
+            <note type="marginal-note" place="margin-right">
+              <lb ed="#Z"/>Divisio signorum 
+              <lb ed="#Z"/>ex <name>Augustino</name>. 
+            </note>
+          </seg>. <quote>Naturalia</quote> vocat, <quote>quae sine 
+          <lb ed="#Z"/>voluntate atque ullo appetitu significandi praeter se aliquid aliud ex se cognosci fa<lb ed="#Z"/>ciunt; 
+          sicuti est fumus, significans ignem. Non enim volens significare id facit</quote>. 
+          <lb ed="#Z"/><quote>Data sunt</quote>, inquit, <quote>quae sibi quaeque viventia invicem dant, ad demonstrandos quan<lb ed="#Z"/>tum 
+          possunt motus animi sui, vel sensa aut intellecta quaelibet.</quote> Et Data quidem 
+          <lb ed="#Z"/>signa rursus distinguit per sensus. Alia enim ad oculos pertinent, ut sunt vexilla 
+          et signa ducum, motus manum et omnium membrorum. Alia ad aures, ut sunt 
+          <lb ed="#Z"/><quote>tuba et tibia</quote>, et ipsa denique verba, quae <quote>inter homines obtinuerunt principatum 
+          <lb ed="#Z"/>significandi</quote>. Ad odorem refert <seg>illum odorem unguenti evangelicum 
+            <note type="marginal-note" place="margin-right">Ioan. 12</note></seg>, quo dominus ali<lb ed="#Z"/>quid 
+          significare voluerit. Ad gustum retulit <seg>Coenam domini 
+            <note type="marginal-note" place="margin-right">Marc. 14</note></seg>. <quote>Sacramento enim san<lb ed="#Z"/>guinis 
+          et corporis sui</quote>, inquit, <quote>praegustato, significavit quod voluit</quote>. Addit et exem<pb ed="#Z" n="322-v"/>plum 
+          <!-- folio 322v -->
+          ad tactum, <quote>Et cum <seg>mulier tangendo fimbriam vestimenti eius
+            <note type="marginal-note" place="margin-left">Matth. 9</note></seg> salva facta est,
+          <lb ed="#Z"/>nonnihil significat.</quote> Sic <name>Augustinus</name> de Signorum generibus et differentia disseruit.
+        </p>
+        <p>
+          <!-- this is Scotus, but not verbatim! -->
```

Comment:
Ok, I have a few different ways that I mark this. If it is quote, obviously we can mark it as a quote. 
If it is a rough paraphrase quote, but still intended by the author as a quote, we can mark it as <quote type="paraphrase">. If it is not intended as a quote, but we as editors know that it is copied, abbreviated, or related to scotus, we can you can use some offset mark up to indicate this.